src/main/java/com/compatibility/Compatibility.java

```diff
@@ -0,0 +1,18 @@
+package com.compatibility;
+
+import com.compatibility.tinkers.ToolBrokenCheck;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import org.jetbrains.annotations.Nullable;
+
+public class Compatibility
+{
+    public static boolean checkMiningCompatibility(@Nullable ItemStack stack, @Nullable String tool)
+    {
+        if (ToolBrokenCheck.checkTinkersBroken(stack, tool))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1126)